Content.Client/Preferences/UI/HumanoidProfileEditor.xaml.cs

```diff
@@ -95,37 +73,33 @@ public sealed partial class HumanoidProfileEditor : Control
         private Button _previewRotateLeftButton => CSpriteRotateLeft;
         private Button _previewRotateRightButton => CSpriteRotateRight;
         private Direction _previewRotation = Direction.North;
-        private EntityUid? _previewDummy;
 
         private BoxContainer _rgbSkinColorContainer => CRgbSkinColorContainer;
         private ColorSelectorSliders _rgbSkinColorSelector;
 
         private bool _isDirty;
-        private bool _needUpdatePreview;
         public int CharacterSlot;
         public HumanoidCharacterProfile? Profile;
-        private MarkingSet _markingSet = new(); // storing this here feels iffy but a few things need it this high up
 
         public event Action<HumanoidCharacterProfile, int>? OnProfileChanged;
 
-        public HumanoidProfileEditor(IClientPreferencesManager preferencesManager, IPrototypeManager prototypeManager,
-            IEntityManager entityManager, IConfigurationManager configurationManager)
+        [ValidatePrototypeId<GuideEntryPrototype>]
+        private const string DefaultSpeciesGuidebook = "Species";
+
+        public HumanoidProfileEditor(IClientPreferencesManager preferencesManager, IPrototypeManager prototypeManager, IConfigurationManager configurationManager)
         {
             RobustXamlLoader.Load(this);
+            _entManager = IoCManager.Resolve<IEntityManager>();
```

Comment:
Why move this dependency to be acquired via IoCManager instead of constructor injection like before?

Reply:
I think I used it for testing as it's not used anymore.